app/controllers/vm_controller.rb

```diff
@@ -20,26 +22,16 @@ def show_list
   private ####
 
   def get_session_data
-    @title          = _("Virtual Machines")
-    @layout         = "vm"
-    @lastaction     = session[:vm_lastaction]
-    @showtype       = session[:vm_showtype]
-    @filters        = session[:vm_filters]
-    @catinfo        = session[:vm_catinfo]
-    @display        = session[:vm_display]
     @polArr         = session[:polArr] || ""           # current tags in effect
     @policy_options = session[:policy_options] || ""
   end
 
   def set_session_data
-    session[:vm_lastaction]   = @lastaction
-    session[:vm_showtype]     = @showtype
-    session[:miq_compressed]  = @compressed unless @compressed.nil?
-    session[:miq_exists_mode] = @exists_mode unless @exists_mode.nil?
-    session[:vm_filters]      = @filters
-    session[:vm_catinfo]      = @catinfo
-    session[:vm_display]      = @display unless @display.nil?
     session[:polArr]          = @polArr unless @polArr.nil?
     session[:policy_options]  = @policy_options unless @policy_options.nil?
   end
+
+  def title
```

Comment:
Can you move the `title` method above `private` please?

(The `respond_to?(:title)` bit in `GenericSessionMixin` won't see it otherwise.)